src/app/base/components/node/OverviewCard/_index.scss

```diff
@@ -13,92 +13,96 @@
     }
 
     .overview-card__cpu {
-      @include pseudo-border(left);
+      @media only screen and (min-width: ($breakpoint-small + 1)){
+        @include pseudo-border(left);
+      };
       grid-area: cpu;
       padding: $spv--large 0 0 0;
-      &::after {
-        top: $spv--large;
-      }
     }
 
     .overview-card__memory {
-      @include pseudo-border(left);
+      @media only screen and (min-width: ($breakpoint-large + 1)){
+        @include pseudo-border(left);
+      };
       grid-area: memory;
       padding: $spv--large 0 0 0;
-      &::after {
-        top: $spv--large;
-      }
     }
 
     .overview-card__storage {
-      @include pseudo-border(left);
+      @media only screen and (min-width: ($breakpoint-small + 1)){
+        @include pseudo-border(left);
+      };
       grid-area: storage;
       padding: $spv--large $sph--large 0 0;
-      &::after {
-        top: $spv--large;
-      }
     }
 
     .overview-card__test-warning {
       grid-area: test-warning;
       padding: $spv--large 0 $spv--large $sph--large;
     }
 
+    .overview-card__test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
+
+    .overview-card__storage-test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
+
+    .overview-card__memory-test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
+
+    .overview-card__cpu-test-text {
+      margin-bottom: 0.1rem;
+      padding-top: calc(0.4rem - 1px);
+    }
```

Comment:
These classes all look like they're setting the same things. Could it be a single class that is applied to all of the columns?

This is also using custom spacing params. Could they be based off the spacing variables instead?

Reply:
The class can definitely be simplified, that's easy enough. As for spacing variables, I guess they could be done as a calculation of them? e.g. `$spv--large * 0.1` for 0.1rem and `$spv--large * 0.4* for 0.4rem

Reply:
I guess that would at least scale with the spacing variables though usually I find that if we're having to use very specific spacing then something else is amiss (i.e. using the wrong class or element or applying the spacing to the wrong element).

I did notice that the element with `overview-card__test-text` also has `u-no-margin--bottom` so the `margin-bottom: 0.1rem;` is being overridden, so you could probably remove the margin-bottom styles.

Also, just looking at the figma file it looks like there might just be a 1 rem gap between these links and the content above where as this looks more like 2.5 rem. If that's correct you might be able to drop the top padding and also the bottom padding on these elements: https://github.com/canonical/maas-ui/pull/4366/files#diff-b7088aac56f8eab6260081604edc7246aa32511490d6b86d8f9c27b1292da145L50.

It could be worth double checking with @lyubomir-popov (if you haven't already chatted about this - I might have missed a conversation about this spacing). Either way feel free to land this with whatever you come up with.

Reply:
I've gone and removed the margin-bottom stuff as well as that top and bottom padding, and it looks like that's allowed me to delete that padding calculation entirely. Amy's asked me to merge this and bios boot mode into a separate branch for QA so they can see it all in one, should finally be able to get this merged after that